metagraph/modules/shared_data/src/main/scala/com/my/dor_metagraph/shared_data/validations/Validations.scala

```diff
@@ -20,7 +24,9 @@ object Validations {
     } yield validatedAddress.productR(validateCheckInLimit).productR(validateIfDeviceDORApi)
 
 
-  def deviceCheckInValidationsL1[F[_] : Async](checkInUpdate: CheckInUpdate): F[DataApplicationValidationErrorOr[Unit]] = {
+  def deviceCheckInValidationsL1[F[_] : Async](
+    checkInUpdate: CheckInUpdate
+  ): F[DataApplicationValidationErrorOr[Unit]] = {
     val validateCheckInLimit = validateCheckInLimitTimestamp(checkInUpdate)
     val validateIfDeviceDORApi = validateIfDeviceIsRegisteredOnDORApi(checkInUpdate)
     Async[F].delay(validateCheckInLimit.productR(validateIfDeviceDORApi))
```

Comment:
Don't put in `delay` after the computation is already done. Put the whole thing in the `delay`